### PR TITLE
Added support for additional labels (multi-tenancy via labels)

### PR DESCRIPTION
**Pull request by cmartin81**
I've implemented ability to add additional labels inside the schema for objects. The implementation is inspired by issue #237 (Auth support for multi-tenant graphs implemented via node labels)

Example of usage:
Use directive @additionalLabels which has 1 parameter 'labels' <array of strings>.
Labels is an array of strings. It also supports string interpolation (See example under) using lodash.template lib. The varible $cypherPrams is also accessible within the string

```
type Movie @additionalLabels(labels: ["u_<%= $cypherParams.userId %>", "newMovieLabel"])  {
    id: String
    name: String
    ...
    ...
    ...
```

When creating, updating or deleting it will add 2 new labels to the query. For this case it will be u_123 and newMovieLabel given that $cypherParams.userId = 123. This way it should be possible to add multi-tenant graph via labels.) 
Labels are only applied on node (not edges)

Tests have been changed and all 110 are passing

### CODE REVIEW

**Comment by cmartin81** [comment]
It is failing since I've updated babel from 6 to version 7.
This line is failing: 
```
node -r babel-register ./example/apollo-server/movies-middleware.js
```

To this line:
```
node -r @babel/register ./example/apollo-server/movies-middleware.js
```

**Comment by cmartin81** [comment]
I'm using this for the last week and it looks like everything works great.
Are there anything I can do to get this PR accepted?

File: package.json [hunk 1]
```diff
@@ -8,16 +8,15 @@
...
     "precommit": "lint-staged",
     "prepublish": "npm run build",
-    "pretest": "npm run build",
-    "test": "nyc --reporter=lcov ava test/unit/ --verbose",
+    "test": "nyc --reporter=lcov ava test/unit/**.test.js --verbose",
     "parse-tck": "babel-node test/helpers/tck/parseTck.js",
     "test-tck": "nyc ava --fail-fast test/tck/",
     "report-coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
-    "test-all": "nyc ava test/ --verbose",
+    "test-all": "nyc ava test/**/*.test.js --verbose",
```
Comment (johnymontana) [inline package.json]: This change misses the tests in `test/unit/neo4j-schema`. What was the reason for the change?

**Comment by johnymontana** [comment]
Hey @cmartin81 - thanks for the PR. 

This is really neat when combined with `cypherParams`. I left just a minor request for changes in the `test-all` npm script, but otherwise, I think this is good to go.

We may want to rethink how this is handled when Neo4j v4.0 is released, which will include new access control features, but since this is useful for your use case I think it will be useful for others as well.

Would you mind adding another PR to the docs showing how to use this? Probably as an [additional item in this file.](https://github.com/grand-stack/grandstack.io/blob/master/docs/neo4j-graphql-js-middleware-authorization.md) Thanks!

File: package.json (hunk 1 above)
Comment (cmartin81) [reply package.json]: This is a syntaks change inside ava due to new version (updated ava from 0.25 to 2.2)

Comment (cmartin81) [reply package.json]: The test/**/*.test.js should include all *.test.js within the directory test (also subdirectories)

**Comment by cmartin81** [comment]
Continuous_Integration_and_Deployment
Here you can see. The failing tests have nothing to do with my work.

File: package.json (hunk 1 above)
Comment (johnymontana) [reply package.json]: Looks like I was seeing inconsistencies handling the glob pattern.

**Comment by johnymontana** [comment]
Sorry about that, that was my fault :-\ 

I've fixed the integration tests and merged this in. Thanks for adding this feature!

**Comment by cmartin81** [comment]
Great! Thanks!
Remember that I have updated the docs and made an PR.
https://github.com/grand-stack/grandstack.io/pull/3

**Pull request merged**